.github/workflows/ci.yml

```diff
@@ -37,8 +37,8 @@ jobs:
     - name: Build
       run: npm run build
 
-    - name: Run Coverage
-      uses: codecov/codecov-action@v3
+#    - name: Run Coverage
+#      uses: codecov/codecov-action@v3
```

Comment:
[question/nit]: Are we just going to leave this comment here?

Reply:
We can remove it for clear code.